### PR TITLE
Create Add-Number.js

### PR DESCRIPTION
**Pull request by Aashutoshbro**
adding number in two way 1. as already given input by the user and another by asking user for the numbers

### CODE REVIEW

File: JavaScript-Programs/Add-Number.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+//add number when user enter on initializing
+........................start...........................
```
Comment (PrajwalCH) [inline JavaScript-Programs/Add-Number.js:2]: Comment this

File: JavaScript-Programs/Add-Number.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+const num2 = 3;
+
+// add two numbers
+const sum = num1 + num2;
+
+// display the sum
+console.log('The sum of ' + num1 + ' and ' + num2 + ' is: ' + sum);
+
+
+........................end........................
```
Comment (PrajwalCH) [inline JavaScript-Programs/Add-Number.js:14]: Comment this

File: JavaScript-Programs/Add-Number.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+//As per the user enter input this is the proper way
+...............start...............
```
Comment (PrajwalCH) [inline JavaScript-Programs/Add-Number.js:17]: Comment here too

File: JavaScript-Programs/Add-Number.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+const num2 = parseInt(prompt('Enter the second number '));
+
+//add two numbers
+const sum = num1 + num2;
+
+// display the sum
+console.log(`The sum of ${num1} and ${num2} is ${sum}`);
+
+
+.........................end........................
```
Comment (PrajwalCH) [inline JavaScript-Programs/Add-Number.js:29]: And here too

Comment (Aashutoshbro) [reply JavaScript-Programs/Add-Number.js:29]: okay

**Pull request merged**